patches.json

```diff
@@ -311,6 +311,11 @@
       "3.2.0": "MAGECLOUD-4407__fix_namespace_vertex_tax__3.2.0.patch"
     }
   },
+  "amzn/amazon-pay-and-login-magento-2-module": {
+    "Set Payment info bug": {
+      "3.4.1": "BUNDLE-2554__set_payment_info_bug_fix__3.4.1"
```

Comment:
Why this fix is only for 3.4.1? 
I still see the same code in 3.4.2 https://github.com/amzn/amazon-payments-magento-2-plugin/blob/3.4.2/src/Payment/Model/OrderInformationManagement.php#L148
Where only 3.4.3 has this fix - https://github.com/amzn/amazon-payments-magento-2-plugin/blob/3.4.3/src/Payment/Model/OrderInformationManagement.php#L147

Reply:
And add extension to the patch file
`BUNDLE-2554__set_payment_info_bug_fix__3.4.1.patch`.

Reply:
3.4.1 is the version that will be bundled with Magento 2.3.5 - we do not at this point bundle 3.4.2 - and given this bug, when we increment the version of Amazon-pay that we bundle in future release, it will be at least 3.4.3 or greater which contains this fix

Reply:
> And add extension to the patch file
> `BUNDLE-2554__set_payment_info_bug_fix__3.4.1.patch`.

fixed